packages/react/src/components/Code/index.tsx

```diff
@@ -41,7 +41,8 @@ Code.displayName = 'Code';
 Code.propTypes = {
   children: PropTypes.string.isRequired,
   language: PropTypes.oneOf(['javascript', 'css', 'html', 'yaml']),
-  className: PropTypes.string
+  className: PropTypes.string,
+  tabIndex: PropTypes.number
```

Comment:
?

Reply:
Yeah, this got stuck in from an old branch and not sure why (guessing not rebasing) - needed to remove this.